federator/config/config.sample.js

```diff
@@ -1,11 +1,14 @@
-const fs = require("fs");
+const dotenv = require('dotenv');
+
+dotenv.config();
```

Comment:
as soon as you introduce environment variables (dot.env) you should also add an example of the required config. (.env.example)

Reply:
Added.